client/components/catalog/FilterTag.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
Please implement scenario with selected tags checked, scrollable and filterable tag menu.